WalletWasabi/JsonConverters/UnblindedSignatureJsonConverter.cs

```diff
@@ -20,20 +20,44 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
 		{
 			JArray arr = JArray.Load(reader);
 
-			string c = arr[0].Value<string>();
-			string s = arr[1].Value<string>();
```

Comment:
IMHO you should test on the length of this field before trying to parse it. If you are serializing in base10, then the length of this field can't possibly be more than 78 chars. (`Log10(2^(8*32))`)